docs/_tutorials/publish-subscribe.md

```diff
@@ -43,27 +40,38 @@ In order to send or receive messages to a Solace message router, you need to kno
   </tr>
   <tr>
     <td>Host</td>
-    <td>String of the form <code>DNS name</code> or <code>IP:Port</code></td>
-    <td>This is the address clients use when connecting to the Solace message router to send and receive messages. For a Solace VMR this there is only a single interface so the IP is the same as the management IP address. For Solace message router appliances this is the host address of the message-backbone.</td>
+    <td>String</td>
+    <td>This is the address clients use when connecting to the Solace messaging to send and receive messages. (See Notes below)</td>
   </tr>
   <tr>
     <td>Message VPN</td>
     <td>String</td>
-    <td>The Solace message router Message VPN that this client should connect to. The simplest option is to use the “default” message-vpn which is present on all Solace message routers and fully enabled for message traffic on Solace VMRs.</td>
+    <td>The Solace message router Message VPN that this client should connect to. </td>
   </tr>
   <tr>
     <td>Client Username</td>
     <td>String</td>
-    <td>The client username. For the Solace VMR default message VPN, authentication is disabled by default, so this can be any value.</td>
+    <td>The client username. (See Notes below)</td>
   </tr>
   <tr>
     <td>Client Password</td>
     <td>String</td>
-    <td>The optional client password. For the Solace VMR default message VPN, authentication is disabled by default, so this can be any value or omitted.</td>
+    <td>The client password. (See Notes below)</td>
   </tr>
 </table>
 
-For the purposes of this tutorial, you will connect to the default message VPN of a Solace VMR so the only required information to proceed is the Solace VMR host string which this tutorial accepts as an argument.
+### Notes
+
+* Finding the connectivity information:
+    * **For DataGo**, this messaging connectivity information is found in the service details in the connectivity tab.
```

Comment:
A screenshot of the service details here could be helpful. I could imagine someone being confused and using the management credentials instead of the messaging ones but a screenshot would make it much easier to avoid that.